cmd/webrtc-load-tester/roles/orchestrator.go

```diff
@@ -277,8 +292,7 @@ func playerJobSpec(args loadTestArguments, region string, viewers int, playbackI
 		Args: []string{
 			"-base-url", args.Playback.BaseURL,
 			"-playback-id", playbackID,
-			// TODO: Support region/node-specific playback by building the playback URL here
-			// "-playback-url", stream.PlaybackURL,
```

Comment:
Does the player role already support this flag?

Reply:
Yep seems like you implemented that already!